### PR TITLE
Bids 1177/number of subscriptions

### PR DESCRIPTION
**Pull request by Eisei24**
### <samp>🤖 Generated by Copilot at b4c340c</samp>

Enhance user dashboard and code quality in `handlers/user.go`. Add a tooltip to show subscription types, use constants and template.HTML for better readability and security, and simplify conditions for rendering dashboard elements.

### CODE REVIEW

File: handlers/user.go [hunk 1]
```diff
@@ -757,7 +816,7 @@ func UserNotificationsCenter(w http.ResponseWriter, r *http.Request) {
 
 		val, ok := validatorMap[sub.EventFilter]
 		if !ok {
-			if (utils.GetNetwork() == "mainnet" && strings.HasPrefix(string(sub.EventName), "monitoring_")) || strings.HasPrefix(string(sub.EventName), utils.GetNetwork()+":"+"monitoring_") {
```
Comment (Eisei24) [inline handlers/user.go:760]: Monitoring events don't have a network prefix.

Comment (D13ce) [reply handlers/user.go:760]: https://github.com/gobitfly/eth2-beaconchain-explorer/blob/b4c340c7c8061b05363ba04e586dab99e37d7b3b/handlers/user.go#L575-L579

This looks like a bug then. The function (`UserUpdateMonitoringSubscriptions`) is not used though so I suggest to remove it together with https://github.com/gobitfly/eth2-beaconchain-explorer/blob/945fa921a93d006bc6113a7a4d3d709e993c8e49/cmd/explorer/main.go#L629.

**Review by D13ce — APPROVED** [review]
lgtm

But please mind https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2395#discussion_r1261006292. Maybe add another commit?

**Comment by Eisei24** [comment]
Obsolete `UserUpdateMonitoringSubscriptions` removed, please have another look.

**Review by D13ce — APPROVED** [review]
Still lgtm :)

**Pull request merged**